doc/ref_model/chapters/chapter06.rst

```diff
@@ -82,21 +181,197 @@ Tenant          +      +           +      +      Create/delete only by Cloud Zon
 Virtual compute +      +           +      +      Create/delete by VDC users with appropriate role. Additional operations would include suspend/unsuspend
 Virtual network +      +    +      +      +      Create/delete by VDC users with appropriate role
 =============== ====== ==== ====== ====== ====== ===========================================================================================================
+>>>>>>> master
```

Comment:
remove

Reply:
Fixed in [f39b95f](https://github.com/cntt-n/CNTT/pull/2931/commits/f39b95f9a9cfbd4d02bbb624817269c76e05707e)